src/client/conditions.py

```diff
@@ -240,35 +273,55 @@ class Type(ExtendableStrEnum):
 
 
 class RejectedAssignmentsCount(ComparableRuleCondition, spec_value=RuleConditionKey.REJECTED_ASSIGNMENTS_COUNT):
-    """How many times this assignment was rejected
+    """The number of rejected assignments of a task suite.
+
+    `RejectedAssignmentsCount` is used with collectors:
+    - [AssignmentsAssessment](toloka.client.collectors.AssignmentsAssessment.md)
     """
 
     value: int
 
 
 class RejectedAssignmentsRate(ComparableRuleCondition, spec_value=RuleConditionKey.REJECTED_ASSIGNMENTS_RATE):
-    """Percentage of how many assignments were rejected from this Toloker out of all checked assignment
+    """A percentage of rejected assignments submitted by a Toloker.
+
+    `RejectedAssignmentsRate` is used with collectors:
+    - [AcceptanceRate](toloka.client.collectors.AcceptanceRate.md)
+
+    See also:
+    - [RejectedAssignmentsCount](toloka.client.conditions.RejectedAssignmentsCount.md) — The number of rejected assignments of a task suite.
     """
 
     value: float
 
 
 class SkillId(IdentityRuleCondition, spec_value=RuleConditionKey.SKILL_ID):
-    """The Toloker no longer meets the specific skill filter
+    """The ID of a changed skill which caused access blocking.
+
+    `SkillId` provides details if [PoolAccessRevokedReason](toloka.client.conditions.PoolAccessRevokedReason.md) condition equals `SKILL_CHANGE`.
```

Comment:
...if the [PoolAccessRevokedReason](toloka.client.conditions.PoolAccessRevokedReason.md) condition...